modules/renter/contractor/editor.go

```diff
@@ -104,9 +104,19 @@ func (he *hostEditor) Close() error {
 }
 
 // Upload negotiates a revision that adds a sector to a file contract.
-func (he *hostEditor) Upload(data []byte) (crypto.Hash, error) {
+func (he *hostEditor) Upload(data []byte) (_ crypto.Hash, err error) {
```

Comment:
To be honest the fact that checking a value after a return is possible in go kind of blew my mind. I noticed this syntax the first time in the contractor's editor.go where it says `// release lock early if function returns an error` and I programmed a minimal example to check that this really works.
`func check() (i int) {
	defer func() {
		fmt.Println(i)
	}()
	i = 0
	return 1
}`
Which prints 1. I would still change it to a simple `return` if you want.